config/main.py

```diff
@@ -1227,6 +1227,92 @@ def validate_patch(patch):
     except Exception as e:
         raise GenericConfigUpdaterError(f"Validate json patch: {patch} failed due to:{e}")
 
+
+def multiasic_validate_single_file(filename):
+    ns_list = [DEFAULT_NAMESPACE, *multi_asic.get_namespace_list()]
+    file_input = read_json_file(filename)
+    file_ns_list = [DEFAULT_NAMESPACE if key == HOST_NAMESPACE else key for key in file_input]
```

Comment:
>HOST_NAMESPACE [](http://example.com/codeflow?start=48&length=14)

what is the diff HOST_NAMESPACE vs DEFAULT_NAMESPACE?

Reply:
HOST_NAMESPACE  is 'localhost'
DEFAULT_NAMESPACE is ''